Patches/PlayerControlPatch.cs

```diff
@@ -2771,6 +2782,22 @@ public static void Postfix(PlayerControl __instance)
                     if (BountyHunter.IsEnable && playerRole.Is(CustomRoles.BountyHunter)) 
                         BountyHunter.OnFixedUpdate(player);
 
+                Pelican.OnFixedUpdate();
+                BallLightning.OnFixedUpdate();
+                BloodKnight.OnFixedUpdate(player);
+                Puppeteer.OnFixedUpdate(player);
+                Shroud.OnFixedUpdate(player);
+                NWitch.OnFixedUpdate(player);
+                Wildling.OnFixedUpdate(player);
+                Spiritcaller.OnFixedUpdate(player);
+                Pitfall.OnFixedUpdate(player);
+                Swooper.OnFixedUpdate(player);
+                Wraith.OnFixedUpdate(player);
+                Chameleon.OnFixedUpdate(player);
+                Spy.OnFixedUpdate(player);
+                Yandere.OnFixedUpdate(player);
+            //    Alchemist.OnFixedUpdate(player);
+
```

Comment:
This code is already outdated because I rewrote `FixedUpdate`

Reply:
.